amida-tech/indaba-client
# INBA-255 Make SelectGroupUsers Stylable

#### What's this PR do?
Reduce Grommet usage, apply BEM classnames, and simply component to facilitate styling

#### Related JIRA tickets:
[INBA-255](https://jira.amida-tech.com/browse/INBA-255)

#### How should this be manually tested?
Load http://localhost:3000/project/101 and check that the group editor still works on the Users tab
Examine SelectGroupUsers.js and FilteredList.js for code quality and BEM naming

#### Any background context you want to provide?
@laceyIrvin We still use List and ListItem from Grommet, so if styling `.filtered-list__list` and `.filtered-list__item` aren't enough you may have to override grommet styling on them

#### Screenshots (if appropriate):
